interfaces/hydra_endpoints/v0/README.md

```diff
@@ -0,0 +1,75 @@
+# `hydra_endpoints`
+
+## Usage
+
+This relation interface describes the expected behavior of provider ([Ory Hydra](https://github.com/canonical/hydra-operator))
+and requirer ([Ory Kratos](https://github.com/canonical/kratos-operator)) sides of relation.
+
+Kratos requires `oauth2_provider` parameter in order to be able to interact with Hydra. This parameter is the Hydra's admin API endpoint,
```

Comment:
This explanation is unnecessary here but can be in the PR description to add context